### ISSUE TITLE
What's New in Python 3.13 (copyediting)

### ISSUE THREAD

**Issue by AA-Turner** [python/cpython#109975]
# Documentation

This is a meta-issue to capture PRs for copyedits to What's New in Python 3.13.

Other meta-issues:

* 3.14: #123299
* 3.12: #109190

### Linked PRs
* gh-110993
* gh-110994
* gh-110997
* gh-114401
* gh-117902
* gh-118439
* gh-118694
* gh-118711
* gh-119958
* gh-119959
* gh-122958
* gh-122971
* gh-122990
* gh-123032
* gh-123063
* gh-123086
* gh-123101
* gh-123132
* gh-123150
* gh-123164
* gh-123292
* gh-123301
* gh-123308
* gh-123393
* gh-123396
* gh-123529
* gh-123552
* gh-123589
* gh-123590
* gh-123776
* gh-123794
* gh-123845
* gh-124135
* gh-124313
* gh-124334
* gh-124336
* gh-124341
* gh-124343
* gh-124348
* gh-124357
* gh-124360
* gh-124362
* gh-124365
* gh-124376
* gh-124827
* gh-124828
* gh-124831
* gh-124830
* gh-124833
* gh-124947
* gh-124966
* gh-125007
* gh-125033
* gh-125034
* gh-127816
* gh-129242
* gh-136046
* gh-137766
* gh-137767
* gh-138370

**Comment by hugovk** [comment]
Let's call out the increased share in full support from 3.13 onwards in [PEP 602](https://peps.python.org/pep-0602/):

https://github.com/python/peps/commit/619e4e239383b5f59e34ebebc36278aebf599a5b

**Comment by hugovk** [comment]
`Py_TRASHCAN_*` - moved in https://github.com/python/cpython/pull/110994.

PEP 602 - added in https://github.com/python/cpython/pull/110997.

**Comment by hugovk** [comment]
When it comes to the [Free-threaded CPython](https://docs.python.org/3.13/whatsnew/3.13.html#free-threaded-cpython) section, I think it can benefit from a bit of a rewrite as it was written fairly early on in development.

---

Right now, it only really talks about how to build your own, but since then we have it as part of the official macOS and Windows installers, so they need to be mentioned.

We have a guide in the docs for installing via Windows, so let's link to this: https://docs.python.org/3.13/using/windows.html#installing-free-threaded-binaries

But not yet for macOS, so ideally this should also be added. There's some info in https://github.com/python/cpython/issues/120098.


<details>
<summary>macOS installer screenshots</summary>

![image](https://github.com/user-attachments/assets/23e8f9d8-bda2-4cb3-9cc1-ed43b4da1281)
![image](https://github.com/user-attachments/assets/90a0f01b-89b8-4c33-a7ab-1e9429490d9b)


</details>

---

It also only mentions `sysconfig.get_config_var("Py_GIL_DISABLED")` for checking if the interpreter is configured for free-threading. We've since added the more-obvious "experimental free-threading build" to the double version flag:

```console
❯ python3.13t --version --version
Python 3.13.0rc1 experimental free-threading build (v3.13.0rc1:e4a3e786a5e, Jul 31 2024, 19:57:27) [Clang 15.0.0 (clang-1500.3.9.4)]
```

**Comment by AA-Turner** [comment]
I should have closed this a while ago as Python 3.13 has been released. Thanks all for the work on *What's New*!

A
